src/bindfs.c

```diff
@@ -173,6 +173,8 @@ static struct Settings {
     uid_t uid_offset;
     gid_t gid_offset;
 
+    long bindfs_size; //Maybe it should be unsigned in future, however right now i use -1 to signal it's not used
```

Comment:
`off_t` is probably appropriate here. It's signed, and since we compile with `-D_FILE_OFFSET_BITS=64` it's also 64 bits.